README.md

```diff
@@ -10,13 +10,41 @@ cargo install stellar-contract-cli --git https://github.com/stellar/stellar-cont
 
 ## Usage
 
+All values passed to `--arg` are the JSON representation of SCVals.
+
+### Directly invoking a function in a user specified WASM contract
+
 ```
-stellar-contract-cli invoke --file <WASMFILE> --fn <FUNCNAME> --arg i32:1 --arg i32:2
+stellar-contract-cli invoke --file <WASMFILE> --fn <FUNCNAME> --arg '{"i32":32}' --arg '{"i32":4}'
 ```
 
 Example using the [example_add_i32](https://github.com/stellar/rs-stellar-contract-sdk/tree/main/examples/add_i32) contract:
 
 ```
-$ stellar-contract-cli invoke --file example_add_i32.wasm --fn add --arg i32:1 --arg i32:2
-i32:3
+$ stellar-contract-cli invoke --file example_add_i32.wasm --fn add --arg '{"i32":32}' --arg '{"i32":4}'
+
+{
+  "i32": 36
+}
+```
+
+### Creating and invoking a contract in storage
+
+#### Creating a contract
+
+```
+Example:
+stellar-contract-cli deploy --id 3d4017c3e843895a92b70aa74d1b7ebc9c982ccf2ec4968cc0cd55f12af4660c --file example_add_i32.wasm
```

Comment:
Is there a way we could support smaller IDs? One thing I appreciated using some other CLI tools in this space was when they would allow me to use invalid values for things like a contract hash, like `0x1`, `0x2`, etc which is helpful in testing. I think within the CLI tool we could simply right-fill with zeros to the 32 byte size we need.

Reply:
Yeah I think this is a great idea. Opened an issue here https://github.com/stellar/stellar-contract-cli/issues/45.

Reply:
https://github.com/stellar/soroban-cli/pull/49